.github/workflows/deploy-feature-branch.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+    branches:
+    - master
+    types:
+      - opened
+      - reopened
+
+name: Initial Feature Branch Deploy
+jobs: 
+  deploy:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '18.x'
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+      - name: Deploy Feature Branch Infrastructure
+        run: |
+          npm ci
+          cd cdk-infra/
+          npm ci
+          npm run deploy:feature -- -c env=dotcomstg -c customFeatureName=enhancedApp-dotcomstg-${{github.head_ref}} --outputs-file outputs.json
```

Comment:
Hmmm, good point. What do you think would be the best environment to deploy to for feature branches?

Reply:
For posterity, this was discussed offline. There was a misunderstanding on my part regarding Slack behavior. Since the Slack bot is currently configured to perform `/test-deploy` with the regular `dotcomstg` Autobuilder by default, deploying features should not directly interfere with existing `/test-deploy` behavior. Having a way to test production deployments for a feature will be handled separately (see PR description for more details)